### ISSUE TITLE
Implement Rule for 700__9#BEARD# 

### ISSUE THREAD

**Issue by kprzerwa** [CERNDocumentServer/cds-dojson#182]
Regarding: 700__9#BEARD#, we could add a "curated_relation" property to:
cds-dojson/cds_dojson/schemas/records/books/ymls/elements/id.yml
When #BEARD#, it curated_relation -> false. 
Is this the meaning of this information?

_Originally posted by @agentilb in https://github.com/CERNDocumentServer/cds-dojson/issues/179#issuecomment-456335412_

**Comment by agentilb** [comment]
Unless we just add a property "Source" that is not mandatory? This is probably easier.

**Comment by kprzerwa** [comment]
it was decided to add a field to an author object not to the id.yml

**Comment by agentilb** [comment]
Ok, but I understood that this information referred to an ID, as per Ludmila's comment: 

_@agentilb 700__9:#BEARD# means that the ID was computed by a script and it might not be 100% correct. I would suggest to keep it, so we know that the ID information should be taken with a grain of salt. Does this seem ok for you?_

**Comment by kprzerwa** [comment]
we had discussed with Ludmila and it was referring to ID of an author not id.yml

**Comment by agentilb** [comment]
Yes, id.yml corresponds to author's ids (ORCID, ccid, CDS, Inspire...)

**Comment by kprzerwa** [comment]
sorry, I though that id.yml is used for other objets too, not just authors. I will review the field and check what will be the best way to migrate it.
